packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/StakeEthModal/StakeEthForm/FractionButtons.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { isZero } from '@suite-common/wallet-utils';
+import { Translation } from 'src/components/suite';
+import { Button } from '@trezor/components';
+import { useStakeEthFormContext } from 'src/hooks/wallet/useStakeEthForm';
+
+const Flex = styled.div`
+    display: flex;
+    gap: 4px;
+`;
+
+const SmallButton = styled(Button).attrs(props => ({
+    ...props,
+    variant: 'tertiary',
+    type: 'button',
+    size: 'small',
+}))``;
+
+export const FractionButtons = () => {
```

Comment:
Extracted to `components/suite`, gave a more generic name, made some props. Now it can be extended and used when needed.

https://github.com/everstake/trezor-suite/pull/3/commits/65033fe49bd94c745b97c9307f73c858cd657081
https://github.com/everstake/trezor-suite/pull/3/commits/1d1721ce124e600239df1fd8da961e4950bf2d3b